### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by s1204IT**
- Codes shaping
- Remove options
  - GApps variant: `OpenGApps`
    - Not worthy of existence
  - Custom Magisk
    - It can be implemented by creating a URL input box, but there is a risk of distributing exceptionally virus-infected files from this repository.
  - Offline mode
    - No need
  - Help
    - No need

At this time, Android 12.1 and earlier cannot be used in this repository, so I deleted unnecessary items.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -33,11 +33,11 @@ jobs:
         with:
           packages: ca-certificates patchelf e2fsprogs aria2 p7zip-full attr unzip xz-utils qemu-utils python3-venv erofs-utils linux-modules-extra-azure
           version: 1.0
-          
+
       - name: Update Package 👷
         working-directory: scripts
         run: |
-          sudo apt-get update -y
```
Comment (YT-Advanced) [inline .github/workflows/build.yml:40]: Why remove this ones. I think we need force yes

**Comment by YT-Advanced** [comment]
@s1204IT I think we shouldn't delete Skip download WSA. We need it for manual WSA URL Link (or we can make our own)

**Comment by s1204IT** [comment]
> @s1204IT I think we shouldn't delete Skip download WSA. We need it for manual WSA URL Link (or we can make our own)

I understand what you mean, but there is a problem.
That is, libraries will no longer be downloaded either.
It'll prevent the build from continuing.
If you keep the library as a file in the repository, it makes no sense to bother with MagiskOnWSA to get the latest version.

**Comment by YT-Advanced** [comment]
> I understand what you mean, but there is a problem.
> That is, libraries will no longer be downloaded either.
> It'll prevent the build from continuing.
> If you keep the library as a file in the repository, it makes no sense to bother with MagiskOnWSA to get the latest version.

We can download VCLib and also WSA from it. You can check test.yml in my branch

**Comment by s1204IT** [comment]
Hi,
--skip-download-wsa will be reverted today.

**Comment by s1204IT** [comment]
The problem of not being able to mount with Actions remains unresolved, so I have opened this pull request.

@YT-Advanced Please review and approve.
But don't merge.
I'll do it.

**Review by YT-Advanced — APPROVED** [review]

**Comment by YT-Advanced** [comment]
> The problem of not being able to mount with Actions remains unresolved, so I have opened this pull request.
> 
> @YT-Advanced Please review and approve.
> But don't merge.
> I'll do it.

So you found a way for this problem ?

**Comment by s1204IT** [comment]
> > The problem of not being able to mount with Actions remains unresolved, so I have opened this pull request.
> > @YT-Advanced Please review and approve.
> > But don't merge.
> > I'll do it.
> 
> So you found a way for this problem ?

No.
Unfortunately I am not clever enough to solve this problem.

**Pull request merged**